fluss-server/src/main/java/com/alibaba/fluss/server/kv/rocksdb/RocksDBResourceContainer.java

```diff
@@ -74,28 +77,25 @@ public class RocksDBResourceContainer implements AutoCloseable {
     /** The handles to be closed when the container is closed. */
     private final ArrayList<AutoCloseable> handlesToClose;
 
+    @Nullable private Statistics statistics;
+    @Nullable private Cache blockCache;
```

Comment:
They are initialized in the constructor, make them `final`, so the getters of them are thread-safe.